recipes/mtbseq/meta.yaml

```diff
@@ -3,11 +3,11 @@ package:
   version: "1.0.3"
 
 source:
-  url: https://github.com/ngs-fzb/MTBseq_source/releases/download/v{{ PKG_VERSION }}/MTBseq_source-{{ PKG_VERSION }}.zip
-  sha256: 965bf19cba5e95bdea653415eedb9d94edd0b4f9e2f3f37ecd2ea266550b353a
+  url: https://github.com/ngs-fzb/MTBseq_source/archive/v{{ PKG_VERSION }}.zip
+  sha256: 916e7df0ede3f09a0013d013b2307d72fc491659a72c60dfc40eeffb3173dca2
```

Comment:
Why did the checksum changes? Is this a different version?

Reply:
Rebuild of the version with a change in the manual. Only relevant for bioconda. A depency needs a after bioconda installation setup.